src/lib/feature-finder.js

```diff
@@ -9,6 +9,7 @@ let promiseGlob = Promise.promisify(glob);
 let gherkinParser = new Gherkin.Parser();
 
 export default function(cucumberOptions) {
+  let didDetectErrors = false;
```

Comment:
i went with did because it implies a boolean. Both errors and detectedErrors could be arrays